src/App.vue

```diff
@@ -205,8 +205,21 @@ th {
 }
 
 /* SIGN IN/SIGN UP/ BASICALLY FORMS LMAO */
-form {
+.container {
   margin: auto;
+  border: solid;
+  width: 450px;
+}
+
+form {
```

Comment:
Another form selector here...we need to make sure that there's only one general form selector.